rust/src/lib.rs

```diff
@@ -1340,7 +1396,7 @@ pub extern "C" fn zcashlc_shield_funds(
     db_data: *const u8,
     db_data_len: usize,
     account: i32,
-    tsk: *const c_char,
+    xprv: *const c_char,
```

Comment:
can UnifiedSpendingKeys be encoded on a string representation we could pass to this method instead of extsk and xpriv?

Reply:
I guess that's for another PR 😬

Reply:
We have intentionally not defined a USK string representation.